### PR TITLE
add prop for alt tag

### PR DESCRIPTION
**Pull request by psimyn**
For https://github.com/Astrocoders/react-image-smooth-loading/issues/6

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ var Img = _react2.default.createClass({
     placeholder: _react.PropTypes.string,
     imgClasses: _react.PropTypes.oneOfType([_react.PropTypes.string, _react.PropTypes.array]),
     holderClasses: _react.PropTypes.oneOfType([_react.PropTypes.string, _react.PropTypes.array]),
-    onClick: _react.PropTypes.func
+    onClick: _react.PropTypes.func,
+    alt: _react.PropTypes.string
```
Comment (fakenickels) [inline dist/index.js:38]: could you put a trailing comma here?

File: dist/index.js [hunk 2]
```diff
@@ -57,11 +58,13 @@ var Img = _react2.default.createClass({
...
         className: (0, _classnames2.default)(_styles2.default['intended'], this.props.imgClasses),
-        src: this.props.src
+        src: this.props.src,
+        alt: this.props.alt
       }),
       _react2.default.createElement('img', {
         className: (0, _classnames2.default)(_styles2.default['placeholder'], this.props.imgClasses),
-        src: this.props.placeholder || Img.globalPlaceholder
+        src: this.props.placeholder || Img.globalPlaceholder,
+        alt: this.props.alt
```
Comment (fakenickels) [inline dist/index.js:67]: a trailing comma also here

**Comment by fakenickels** [comment]
Awesome! Thanks for the PR, left some comments on the diff.

**Comment by psimyn** [comment]
@grsabreu those were generated - my only change was to [`lib/index.js`](https://github.com/Astrocoders/react-image-smooth-loading/pull/7/files#diff-6d186b954a58d5bb740f73d84fe39073R18) which has trailing comma

Wasn't sure if I should commit generated files or if that only happened pre-release

**Comment by fakenickels** [comment]
haha true! @psimyn I was super tired when I look and don't paid attention.

**Pull request merged**